proto/api.proto

```diff
@@ -721,12 +732,27 @@ message GetAccountBalanceResponse {
 
 message TokenBalance {
   string symbol = 1;
-  string address = 2;
-  double walletAmount = 3;
+  string tokenMint = 2;
+  double settledAmount = 3;
   double unsettledAmount = 4;
   double openOrdersAmount = 5;
 }
 
+message GetTokenAccountsRequest {
+  string ownerAddress = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message GetTokenAccountsResponse {
+  repeated TokenAccount accounts = 1;
+}
+
+message TokenAccount {
+  string symbol = 1;
+  string tokenMint = 2;
+  string tokenAccount = 3;
+  double settledAmount = 4;
```

Comment:
think should just call this one `amount`, since there's no concept of settled/unsettled for token accounts